components/partials/portfolio/balances.vue

```diff
@@ -299,10 +299,10 @@ export default Vue.extend({
           return list
         }
 
-        const quoteToken = market.oracleQuote.toLowerCase()
+        const quoteDenom = market.oracleQuote.toLowerCase()
```

Comment:
I was asking about this: `market.oracleQuote.toLowerCase()`, why are we not using `market.quoteDenom.toLowerCase()`